python/GafferSceneUI/ShufflePrimitiveVariablesUI.py

```diff
@@ -49,7 +49,7 @@
 
 	An additional context variable \"${source}\" can be used on the destination plugs to insert
 	the name of each source primitive variable. For example, to prefix all primitive variables
-	with \"user:\" set the source to \"*\" and the destination to \"user:${source}\").
+	with \"render:\" set the source to \"*\" and the destination to \"render:${source}\").
```

Comment:
These are primvars, where prefixes don't really matter. Might be best to clarify that by not using colon-based prefix, or by using something other than `user:` or `render:`.

Reply:
I _think_ a common use case here is to promote primvars as attributes via an Instancer, which is why I wrote the original docs this way. Also, I found it easier to reason about the `${source}` behaviour via some prefixed name (same reason I did that in the tests originally). Happy to change it so something different if you prefer though. This was just a search/replace `user:` for `render:` really...

Reply:
That may have been true at some point, but the Instancer does that prefixing automatically now. I agree that a common use of `${source}` is to prefix/suffix things, so it makes sense to use that as an example. But this attribute/primvar nomenclature mapping is confusing enough already, so we need to do everything we can to avoid muddying the waters.

I'm trying to think of a primitive-variable-specific example, but the closest I'm getting is `P->Pref and N->Nref`, and those seem easier just to do explicitly. Still, seems like `${source}ref` might get the idea across?